src/v3/test/integration/util/createAuthJsPayload.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+type MethodType = 'POST' | 'GET';
+
+export const createAuthJsPayloadArgs = (
+  methodType: MethodType,
+  uri: string,
+  data?: Record<string, unknown>,
+  acceptVersion = 'application/json; okta-version=1.0.0',
+  contentType = 'application/json',
```

Comment:
Curious if we ever do override these defaults (sorry, I didn't search for it since there's too many test changes to comb through)

Reply:
Yea, we do :( [this](https://github.com/okta/okta-signin-widget/pull/2777/files#diff-44074d6a427e46e6968f87fb2f4456fa4facf206d5f4387339f862edc2a40a89R57) is one example (of a couple), I looked into changing the test file to match but this mock file originates from v2 (so decided it would not be a good idea to change).

Reply:
Okay thanks for pointing the usage out - Not a big deal, just wanted to make sure we needed the overridability.